lib/ui/navbar.js

```diff
@@ -363,19 +372,46 @@ export default class IpNavBar extends React.Component {
         "connecting": true,
       });
       SignalkClient.on('connect', () => {
-        SignalkClient.subscribe({context:'vessels.self',subscribe:[{ path:'*'}]});
-        SignalkClient.on('delta', this.props.instrumentPanel.dispatch.bind(this.props.instrumentPanel));
+        // For debug only with npm start
+        var token = retrieveToken();
+        if (token) {
+          debug("[connectOrDisconnect] connect with token authentication");
+          SignalkClient.setAuthenticated(token);
+        }
+        // End for debug only
+        debug("[connectOrDisconnect] Waiting for connectionInfo");
+        SignalkClient.on('connectionInfo', (data => {
+          debug("[connectOrDisconnect] connectionInfo received");
+          if (!isPreferredUnits() && this.props.instrumentPanel.getSsAvailable()) {
+            debug("[connectOrDisconnect] storage server is available & no local preferred units");
```

Comment:
applicationData is the name of the feature on the server, so use it here also?

Reply:
Changed to `applicationData`